tensorflow/java/src/main/java/org/tensorflow/types/TFBool.java

```diff
@@ -0,0 +1,14 @@
+// GENERATED FILE. Edit tftypes.pl instead.
+package org.tensorflow.types;
+
+/** The class TFBool represents Boolean. */
+public class TFBool implements Types.TFType {
+  /** Represents the type TFBool at run time. */
+  public static final Class<TFBool> T = TFBool.class;
```

Comment:
I wonder if there is value in keeping a level of abstraction here. We might later decide to use something other than the class object as the representative of the type. If we call it "class" then we lose that flexibility.